File: step0055/app/src/main/java/de/conciso/starter/HelloWorldApplication.java
package de.conciso.starter;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class HelloWorldApplication {

  private static final Logger log = LoggerFactory.getLogger(HelloWorldApplication.class);

  private final Greeter greeter;

  public HelloWorldApplication(Greeter greeter) {
    this.greeter = greeter;
  }

  public void run() {
    log.info(greeter.greet("World"));
  }
}

File: python0060/hello_world_application.py
import logging

from greeter import Greeter


class HelloWorldApplication:
    _log = logging.getLogger(__qualname__)

    def __init__(self, greeter: Greeter) -> None:
        self._greeter = greeter

    def run(self) -> None:
        message = self._greeter.greet("World")
        self._log.info(message)
